README.md

```diff
@@ -1 +1,12 @@
-# sdcore-bundles
\ No newline at end of file
+# sdcore-bundles
```

Comment:
Would this repo be a good place to have an index of all the operators for sdcore? Similar to the [`charmed-magma`](https://github.com/canonical/charmed-magma) repo.

Reply:
I believe it's great to have an entry point for the docs, and all the code referenced in one place. However, can defer this work when building the docs, and we might have a better place at that time, or we won't even need it.